### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by mehanalavimajd** [pashmaklang/pashmak#66] [Ideas]
I think we have to create a website for pashmak . 
If you want I can make it in pashmak.github.io

**Comment by parsampsh** [comment]
Yes. i'm waiting for #32

**Reply by mehanalavimajd** [reply]
So I can create a new repository and name it pashmaklang.github.io then It will make a new website

**Reply by mehanalavimajd** [reply]
And I can use  pashmak html in it

**Reply by parsampsh** [reply]
Thanks. this is perfect. If you wanna do this, create repo under pashmaklang org.

**Reply by mehanalavimajd** [reply]
No we can only choose pashmaklang.github.io because the github will give free host and free domain

**Reply by parsampsh** [reply]
I'm sorry i means that create repo in pashmaklang organization(i not mean pashmaklang.org (: ). `pashmaklang/pashmaklang.github.io` :)
